kotsadm/pkg/airgap/update.go

```diff
@@ -95,12 +95,21 @@ func UpdateAppFromAirgap(a *app.App, airgapBundle multipart.File) (finalError er
 	if err != nil {
 		return errors.Wrap(err, "failed to create temp file")
 	}
+
+	if err := task.SetTaskStatus("update-download", "Copying package files...", "running"); err != nil {
```

Comment:
how about `Copying package`? it feels weird to say `Copying files` before extracting them.